spec/models/work_package/pdf_export/common/macro_spec.rb

```diff
@@ -47,48 +93,399 @@
     end
   end
 
-  describe "wp mention tag" do
-    let(:markdown) { '<mention class="mention" data-id="185" data-type="work_package" data-text="#185">#185</mention>' }
+  describe "wp mention macro" do
+    describe "with tag" do
+      let(:markdown) { '<mention class="mention" data-id="185" data-type="work_package" data-text="#185">#185</mention>' }
+
+      it "ignores the tag" do
+        expect(formatted).to eq("<mention class=\"mention\" data-id=\"185\" " +
+                                "data-type=\"work_package\" data-text=\"#185\">\\#185</mention>")
```

Comment:
Cleaner with single quotes and no escaping in this and other places
```suggestion
        expect(formatted).to eq('<mention class="mention" data-id="185" ' +
                                'data-type="work_package" data-text="#185">#185</mention>')
```

Reply:
changed everywhere except where there is now a template string
https://github.com/opf/openproject/pull/19449/commits/81efe18f33001302d82d8b6c55fa808ecda766f9